src/components/Verse/VerseText.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+import VerseType from '../../../types/VerseType';
+import QuranWord from '../dls/QuranWord/QuranWord';
+
+type VerseTextProps = {
+  verse: VerseType;
+  fontStyle?: 'uthmani' | 'madani' | 'indopak';
```

Comment:
feels like this string literal should be a type somewhere? 
```ts
type FontStyleOptions = 'uthmani' | 'madani' | 'indopak';
```

Reply:
Yepppp, made it an enum and refactored the code to use it